<launch>
	<arg name="gui" default="False" />
	<param name="robot_description" command="$(find xacro)/xacro.py '$(find jaco_model)/urdf/standalone_arm.xacro'" />
	<param name="use_gui" value="$(arg gui)"/>
	<node name="robot_state_publisher" pkg="robot_state_publisher" type="state_publisher" />
	<node name="rviz" pkg="rviz" type="rviz" args="-d $(find jaco_model)/jaco_model.rviz" required="true" />

	<!-- Added by Eric -->
	<node name="move_arm" pkg="jaco_model" type="move_arm" output="screen"/>
	<node name="jaco_plus_reflex_demo" pkg="jaco_model" type="jaco_plus_reflex_demo.py" output="screen" required="true" />  
</launch>

